contrib/README.md

```diff
@@ -0,0 +1,36 @@
+This folder contains contributed files for use with Sopel
+
+## Service configuration
+
+`sopel.service` and `sopel.cfg` designed to be distributed by third parties
+such as Fedora Project or Arch Linux.
+
+`sopel.cfg` is a default configuration file for sopel, that assumes the OS is
```

Comment:
This was wrong in the plain-text file, but obviously never fixed because `contrib/` maintenance attitude is "YOLO".

```suggestion
`sopel.cfg` is a default configuration file for Sopel, that assumes the OS is
```